frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -101,21 +101,21 @@ comments-Comment--unpin-button = UNPIN
 
 editor-EditorMenu--sign-in-to-translate = <a>Sign in</a> to translate.
 editor-EditorMenu--read-only-localization = This is a read-only localization.
-editor-EditorMenu--button-copy = Copy
+editor-EditorMenu--button-copy = COPY
     .title = Copy From Source (Ctrl + Shift + C)
-editor-EditorMenu--button-clear = Clear
+editor-EditorMenu--button-clear = CLEAR
     .title = Clear Translation (Ctrl + Shift + Backspace)
-editor-EditorMenu--button-approve = Approve
+editor-EditorMenu--button-approve = APPROVE
     .title = Approve Translation (Enter)
-editor-EditorMenu--button-approving = <glyph></glyph>Approving
+editor-EditorMenu--button-approving = <glyph></glyph>APPROVING
```

Comment:
I just noticed that the glyph (spinner) never appears when saving or approving translation. It seems unrelated to this patch, though. Can somebody confirm? We should file a bug.

Reply:
Everything else is done I'm just going to wait on answer on this specific problem to see if I can do anything  about it right.

Reply:
@Mitch4sho I'll test this out when I have a chance to review this patch and update if there is something that needs/can be done in this patch or file a bug accordingly.

Reply:
@mathjazz The spinners should appear on this button as well as `Suggesting` and `Saving` but it does not appear that they are. What is interesting is that there are tests for both `Saving` and `Suggesting` that check for the class `.fa-spin` that are not failing. So, that leads me to believe that the class is being applied, but the spinner is not actually rendering. 

It looks like those were [added](https://github.com/mozilla/pontoon/pull/1564) a little while back, but are no longer working??  Would it be better to [reopen the original bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1614588) or file a new one?

Reply:
Thanks. Reopened https://bugzilla.mozilla.org/show_bug.cgi?id=1614588 and added a comment with my findings.